bin/format_code.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+for i in `find src -type f -name '*.rs'`
+	do rustfmt --write-mode=overwrite $i
+done
```

Comment:
for handling whitespace, etc correctly:
```bash
find src -type f -name '*.rs' -print0 | while read -d $'\0' -r f; do
  rustfmt --write-mode=overwrite "${f}"
done
```
not that we're likely to need this here, but still.